application/forms/EventRuleConfigElements/EscalationCondition.php

```diff
@@ -254,16 +245,15 @@ public function hasValue(): bool
     public function getCondition(): string
     {
         $filter = Filter::any();
-        /** @var int $count */
-        $count = $this->getValue('condition-count');
-        $removePosition = $this->getValue('remove');
+        $count = (int) $this->getValue('condition-count');
+        $removePosition = (int) $this->getValue('remove');
         if ($removePosition) {
             $count += 1;
         }
 
         if ($count > 0) { // if count is 0, loop runs in reverse direction
```

Comment:
You can return earlier in this case.
```suggestion
public function getCondition(): string
    {
        $count = (int) $this->getValue('condition-count');
        if ($count === 0) {
            return '';
        }
        
        $filter = Filter::any();
        ...
```